docs/AA--Javascript-in-EE/components/form/select/datetime-select.md

```diff
@@ -0,0 +1,150 @@
+# `<DatetimeSelect />`
+
+DatetimeSelect exposes the [`react-select`](https://deploy-preview-2289--react-select.netlify.com/home) component and generates a select input for choosing from the provided EE Datetime model data. It is a wrapper surrounding the [`<ModelEnhancedSelect />`](model-select.md) higher order component.
+
+The following is the shape of the component with its default props:
+
+```jsx
+<Datetime
+    selectConfiguration={
+        {
+            loadingMessage: () => 'Retrieving Datetimes',
+            noOptionsMessage: () => 'No Datetimes.',
+            placeholder: 'Select Datetime...'
+        }
+    }
+    queryData={
+        {
+            limit: 100,
+            orderBy: 'start_date',
+            order: 'DESC',
+            showExpired: false
+        }
+    }
+    getQueryString={ datetimeModel.getQueryString }
+    selectLabel={ 'Select Datetime' }
+    addAllOptionLabel={ 'All Datetimes' }
+/>
+```
+
+The `DatetimeSelect` component also has one `state` property: `modelName`.  This is set to `datetime`.  It is passed on to `ModelEnhancedSelect` and informs what data is being queried.
+
+## Props
+
+Along with the below props, this component passes through any props usable by the [`<ModelEnhancedSelect />`](model-select.md) component.
+
+### selectConfiguration
+
+This is an object that can contain any values components higher in the tree want to pass down to the `react-select` component.  You can read about all the possible values that can be passed down via [this document](https://deploy-preview-2289--react-select.netlify.com/props).
+
+- Type: `Object`
+- Required: No
+- Default:
+```js
+{
+    loadingMessage: 'Retrieving Datetimes.',
+    noOptionsMessage: 'No Datetimes.'
+    placeholder: 'Select Datetime...'
+}
+```
+
+### queryData
+
+Contains default properties used for the Datetime model REST request.
+
+- Type: `Object`
+- Required: No
+- Default: `{ limit: 100, orderBy: 'start_date', order: 'DESC', showExpired: false }`
+
+### getQueryString
+
+This is a function that returns the queryString used to retrieve the Datetime model data. It defaults to [datetimeModel.getQueryString](../../../../../assets/src/data/model/datetime/index.js).
+
+- Type: `Function`
+- Required: No
+- Default: `datetimeModel.getQueryString`
+
+### selectLabel
+
+If provided, a `<label>` will be prepended before the Select element using the text provided by the value on this property.  It will be linked (via the label `for` attribute) to the select input via the `selectConfiguration.name` property.
+
+- Type: `String`
+- Required: No
+- Default: `'Select Datetime'`
+
+### addAllOptionLabel
```

Comment:
Because `selectConfiguration` is only the props passed on to the `react-select` component.  `selectLabel` and `addAllOptionLabel` are props specific to _our_ components and get transformed to the appropriate settings for the `selectConfiguration`.